frontend/src/TourForm/TourForm.tsx

```diff
@@ -91,7 +90,7 @@ function TourForm({ tour, muralsData, handleCancel }: ITourFormProps) {
   /**
    * Enable editable form fields for admin users
    */
-  const userContext = useContext(Context)
+  const userContext: any = useContext(Context)
   useEffect(() => setIsAdmin(!!(userContext as any).user), [userContext]);
```

Comment:
were the `:any`s added intentionally? if so they should simplify the next line to just `... !!userContext.user ...`

Reply:
If I don't add the any I get an error on userContext.token.i